server/src/test/java/org/opensearch/extensions/ExtensionsOrchestratorTests.java

```diff
@@ -354,6 +363,233 @@ public void testRegisterHandler() throws Exception {
 
     }
 
+    private static class Example implements NamedWriteable {
+        public static final String INVALID_NAME = "invalid_name";
+        public static final String NAME = "example";
+        private final String message;
+
+        Example(String message) {
+            this.message = message;
+        }
+
+        Example(StreamInput in) throws IOException {
+            this.message = in.readString();
+        }
+
+        @Override
+        public void writeTo(StreamOutput out) throws IOException {
+            out.writeString(message);
+        }
+
+        @Override
+        public String getWriteableName() {
+            return NAME;
+        }
+
+        @Override
+        public boolean equals(Object o) {
+            if (o == null || getClass() != o.getClass()) {
+                return false;
+            }
+            Example that = (Example) o;
+            return Objects.equals(message, that.message);
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(message);
+        }
+    }
+
+    public void testGetNamedWriteables() throws Exception {
+        Path extensionDir = createTempDir();
+        List<String> extensionsYmlLines = Arrays.asList(
```

Comment:
Let's initialize this in the test setup. You can remove `extensionsYmlLines` from the other method done if initialized globally.

Reply:
Great idea, I'll make that change